Tests/Domain/PermissionGrantingStrategyTest.php

```diff
@@ -74,7 +76,7 @@ public function testIsGrantedFallsBackToParentAcesIfNoLocalAcesAreApplicable()
 
     public function testIsGrantedReturnsExceptionIfNoAceIsFound()
     {
-        $this->expectException(\Symfony\Component\Security\Acl\Exception\NoAceFoundException::class);
+        $this->expectException(NoAceFoundException::class);
```

Comment:
Shall we move this down just before `$strategy->isGranted(`? (I believe we started doing this in the monorepo)

Reply:
that would make the test more precise about where it expects the exception (putting them at the beginning is the direct migration from the annotation, as the annotations were also expecting the exception anywhere in the test).
So I would say it is a good idea to do it (and here, the testsuite is small enough to allow applying that on existing tests rather than only new ones, if you have time for it)

Reply:
I've fixed this for all test cases that I've touched with this PR.